client/overview/index.js

```diff
@@ -72,7 +72,10 @@ const OverviewPage = () => {
 	} );
 	const tasks =
 		Array.isArray( tasksUnsorted ) && tasksUnsorted.sort( taskSort );
+
 	const queryParams = getQuery();
+	const accountRejected =
+		accountStatus.status && accountStatus.status.startsWith( 'rejected' );
```

Comment:
No change to this variable, just moved above the `show...` variables to follow some sort of order to these declared vars.